steps/unity-install/0.1.0/step.yml

```diff
@@ -0,0 +1,102 @@
+title: Unity Install
+summary: |
+  Installs Unity packages
+description: |
+  Install Unity packages
+website: https://github.com/bdrelling/bitrise-step-unity-install
+source_code_url: https://github.com/bdrelling/bitrise-step-unity-install
+support_url: https://github.com/bdrelling/bitrise-step-unity-install/issues
+published_at: 2019-03-04T19:48:11.697942-06:00
+source:
+  git: https://bdrelling@github.com/bdrelling/bitrise-step-unity-install.git
+  commit: 63ce02bbaf1443921cd44f382cbf3feac9647875
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- xamarin
+type_tags:
+- installer
+toolkit:
```

Comment:
Can remove `toolkit` as step.sh is used by default if not specified.

Reply:
I assume then that `bash` is the default as well? Removing that section.